### PR TITLE
Bug fixes and changes

### PR DESCRIPTION
**Pull request by adityamulik**
Below changes have been implemented in this PR.

- Updated the requirements.txt file
- Updated the shell scripts path for execution (Removed static path from shell scripts)
- Removed group_vars as they weren't being detected by the playbook
- Update the vars in defaults and assertions
- Updated documentation for both roles and added documentation for shell script creation
- Removed static path of /tools/scripts/l18n/pre_translation.sh & /tools/scripts/l18n/post_translation.sh from the tasks and made a var with defaulted values, user can override the path for the script.
- Removed translation lang values from default.

### CODE REVIEW

File: roles/post_translation/README.md [hunk 1]
```diff
@@ -28,11 +29,25 @@ fail)
 Dependencies
 ------------
 
-The pre_translation role should already be executed for the latest strings to be translated and be available on Memsource client for post_translation script to work.
-Global vars are declarared in the root folder as global_vars/all which can be overridden by passing vars from command-line.
+- The pre_translation role should already be executed for the latest strings to be translated and be available on Memsource portal for post_translation script to work.
+- Global vars are declarared in the defaults folder as main.yml which can be overridden by passing vars from command-line or extra_vars.yml file in the collection's root folder.
+
+Steps to create post_translation shell Script
```
Comment (rooftopcellist) [inline roles/post_translation/README.md:35]: :tada: Thank you for adding docs for this!

**Review by rooftopcellist — APPROVED** [review]
I left a comment with one minor nitpick, but once fixed, feel free to merge this.  Thanks!

File: roles/post_translation/README.md [hunk 2]
```diff
@@ -28,11 +29,25 @@ fail)
 Dependencies
 ------------
 
-The pre_translation role should already be executed for the latest strings to be translated and be available on Memsource client for post_translation script to work.
-Global vars are declarared in the root folder as global_vars/all which can be overridden by passing vars from command-line.
+- The pre_translation role should already be executed for the latest strings to be translated and be available on Memsource portal for post_translation script to work.
+- Global vars are declarared in the defaults folder as main.yml which can be overridden by passing vars from command-line or extra_vars.yml file in the collection's root folder.
```
[the remark this answers is not in the document]
Comment (adityamulik) [reply roles/post_translation/README.md]: Fixed as per cdcd5d3d61b70259f0c859e8458ec379eacde2d3

File: roles/pre_translation/README.md [hunk 3]
```diff
@@ -24,10 +25,18 @@ Below are mandatory variables required.
 Dependencies
 ------------
 
-Global vars are declarared in the root folder as global_vars/all which can be overridden by passing vars from command-line.
+Global vars are declarared in the defaults folder as main.yml which can be overridden by passing vars from command-line or extra_vars.yml file in the collection's root folder.
```
Comment (rooftopcellist) [inline roles/pre_translation/README.md:28]: small typo here - `declared`

Comment (adityamulik) [reply roles/pre_translation/README.md]: fixed in the latest push

File: extra_vars.yml [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
 repo_branch: ""
 fork_repo_url: ""
+shell_script_path: ""
 project_name: ""
-project_template: 
\ No newline at end of file
+project_template: 
+memsource_username: ""
+memsource_password: ""
+translation_supported_langs:
```
Comment (rooftopcellist) [inline extra_vars.yml:13]: The user _must_ provide the `translation_supported_langs`. We assert that this is set [here](https://github.com/ansible/ansible-collection-memsource/blob/main/roles/pre_translation/tasks/main.yml#L9).

**Comment by rooftopcellist** [comment]
Feel free to merge this when you feel it is ready @adityamulik

**Pull request merged**